src/Tasks/LDAPMemberSyncTask.php

```diff
@@ -28,7 +28,7 @@ class LDAPMemberSyncTask extends BuildTask
      * @var array
      */
     private static $dependencies = [
-        'ldapService' => '%$' . LDAPService::class,
+        'LDAPService' => '%$' . LDAPService::class,
```

Comment:
This doesn't match the actual `protected $ldapService` and usage within the class (`$this->ldapService->x()` - shouldn't we change this back to be `'ldapService' => xxx`?

Reply:
It needs to match the setter or the private property (whichever exists) eg (set)LDAPService()